datafeeder/frontend-config.json

```diff
@@ -23,5 +23,14 @@
       "value": "EPSG:3857"
     }
   ],
-  "thesaurusUrl": "https://georchestra.mydomain.org/geonetwork/srv/api/registries/vocabularies/search?type=CONTAINS&thesaurus=external.theme.httpinspireeceuropaeutheme-theme&rows=200&q=${q}&uri=**&lang=${lang}"
+  "thesaurusUrl": "https://georchestra.mydomain.org/geonetwork/srv/api/registries/vocabularies/search?type=CONTAINS&thesaurus=external.theme.httpinspireeceuropaeutheme-theme&rows=200&q=${q}&uri=**&lang=${lang}",
+  "links": [
+    {
+      "uri": "/datahub/dataset/:uuid",
+      "title": {
+        "default": "See data in Datahub",
```

Comment:
Considering this is a bit redundant with the above sentence...
![image](https://github.com/user-attachments/assets/8ceebd8f-172b-4fbe-be6b-40ef05bcd058)

I think "datahub" would just be perfect here by default. (same in French)